src/integrations/expo-dev-menu.ts

```diff
@@ -4,16 +4,25 @@ import { store } from '../state';
  * NOTE: this will override any other custom items, not good!
  */
 export const registerExpoDevMenuItem = async () => {
+  let ExpoDevMenu;
   try {
-    const ExpoDevMenu = await import('expo-dev-menu');
+    ExpoDevMenu = await import('expo-dev-menu');
+  } catch {
+    console.log('ScreenSizer: skipped registration of item in expo dev menu.');
```

Comment:
it will always log on a bare react-native project ?
What do you think about throwing an error ?